src/app/people/page.tsx

```diff
@@ -1,5 +1,200 @@
-import { MEMBERS } from '@/data/members'
+'use client'
+
+import styled from '@emotion/styled'
+import { useState, useEffect } from 'react'
+//import { usePathname, useSearchParams } from 'next/navigation'
+import { MEMBERS_KEY, MEMBERS, CareerTypes } from '@/data/members'
+import { Sections, Section, SectionTitle } from '@/components/Section'
+import MemberCard from '@/components/MemberCard'
+import AlumniCard from '@/components/AlumniCard'
+import { Color } from '@/app/theme'
+
+const SectionContent = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: 36px;
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(5, 1fr);
+  }
+`
+const MainInner = styled.div`
+  display: flex;
+`
+
+const AlumniSectionContent = styled.div`
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
+  gap: 1px;
+
+  @media (min-width: 1200px) {
+    grid-template-columns: repeat(5, 1fr);
+  }
+`
+
+const TableOfContents = styled.ul`
+  position: sticky;
+  top: 100px;
+  gap: 12px;
+  display: flex;
+  flex-direction: column;
+
+  margin: 0;
+
+  width: fit-content;
+  height: fit-content;
+`
+
+const AnchorLi = styled.li`
+  list-style-type: none;
+`
+
+const Anchor = styled.a<{ selected?: boolean }>`
+  text-decoration: none;
+  color: ${props => (props.selected ? Color.orange900 : Color.gray900)};
+
+  &:hover {
+    color: ${Color.orange900};
+  }
+`
+
+const SubCategoryTitle = styled.h2`
+  font-size: 1rem;
+  font-weight: bold;
```

Comment:
```suggestion
${FontVariant.title_sm}
```

For consistency, we need to use the pre-defined styles as much as possible. Please don't define any new font styles at least it is required for exceptional reasons.